assets/app/actions/routeActions.js

```diff
@@ -0,0 +1,12 @@
+import { navigationTypes } from '../constants';
+import store from '../store';
+
+export default {
+  redirect(path) {
+    store.dispatch({
+      type: navigationTypes.UPDATE_ROUTER,
```

Comment:
I'm going to start pushing for action creators soon to encapsulate everything that makes a valid action. We don't have any yet, but this is a good place to start bringing it up. It's nice to have one place that tells you, "Give me a path and I'll give you a proper redux action for updating the router." This is also a separate concern from actually choosing the right action to create and send to the store for dispatch.